electrumx_rpc.py

```diff
@@ -58,7 +58,7 @@ def main():
     args = parser.parse_args()
 
     if args.port is None:
-        args.port = int(environ.get('ELECTRUMX_RPC_PORT', 8000))
+        args.port = int(environ.get('RPC_PORT', 8000))
```

Comment:
This is of course unrelated to the change, but in `env` `RPC_PORT` is used instead of `ELECTRUMX_RPC_PORT`.